apps/editing-toolkit/editing-toolkit-plugin/help-center/class-wp-rest-help-center-jetpack-search-ai.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * WP_REST_Help_Center_Jetpack_Search_AI file.
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE;
+
+use Automattic\Jetpack\Connection\Client;
+
+/**
+ * Class WP_REST_Help_Center_Jetpack_Search_AI.
+ */
+class WP_REST_Help_Center_Jetpack_Search_AI extends \WP_REST_Controller {
+	/**
+	 * WP_REST_Help_Center_Jetpack_Search_AI constructor.
+	 */
+	public function __construct() {
+		$this->namespace = 'help-center';
+		$this->rest_base = '/jetpack-search/ai';
+	}
+
+	/**
+	 * Register available routes.
+	 */
+	public function register_rest_route() {
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/search',
+			array(
+				'methods'             => \WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'get_search_results' ),
+				'permission_callback' => array( $this, 'permission_callback' ),
+			)
```

Comment:
```suggestion
				'permission_callback' => array( $this, 'permission_callback' ),
				'args' => array( 'query' => ...
			)
```
Probably a good idea to add `args` declaration of the parameters this endpoint supports. Especially since at the very least `query` is required. Although, I get that it feels kinda like code duplication between this and the WPCOM backend's declaration :/